draft-ietf-intarea-provisioning-domains.md

```diff
@@ -909,42 +940,86 @@ of the additional information has passed).
 
 # Security Considerations {#security}
 
+Since the PvD ID RA option can contain an RA header and other RA options,
+any security considerations that apply for specific RA options continue to
+apply when used within a PvD ID option.
+
 Although some solutions such as IPsec or SeND {{?RFC3971}}
 can be used in order to secure the IPv6 Neighbor
 Discovery Protocol, in practice actual deployments largely rely on link
 layer or physical layer security mechanisms (e.g., 802.1x {{IEEE8021X}})
 in conjunction with RA Guard {{?RFC6105}}.
 
+Fragmentation of RAs can lead to processing only part of a PvD ID
```

Comment:
Humm there is a RFC saying that fragmented ND packets should be ignored.

Reply:
Rather than fragmented, I meant the multiple RAs sent to avoid fragmentation (the "split" RAs")

Reply:
Updated this text!